wp-modules/app/js/src/components/App/index.tsx

```diff
@@ -44,10 +48,18 @@ export default function App() {
 			<Header />
 			<Patterns
 				Notice={
-					<VersionControlNotice
-						isVisible={ versionControl.displayNotice }
-						handleDismiss={ versionControl.updateDismissedThemes }
-					/>
+					<div className="patternmanager-notice-container">
+						<EnvironmentNotice
+							isVisible={ environment.displayNotice }
+							handleDismiss={ environment.updateDismissedSites }
+						/>
+						<VersionControlNotice
```

Comment:
This looks good.

If we ever add a 3rd, let's add some abstraction for them.

But 2 separate ones is good.

Reply:
Sounds like a plan!